### PR TITLE
ELE-1206 bug when upgrading elmentary package after metrics tables schema is changed

### PR DESCRIPTION
**Pull request by IDoneShaveIt**

### CODE REVIEW

File: macros/edr/tests/test_utils/create_elementary_test_table.sql [hunk 1]
```diff
@@ -3,6 +3,15 @@
         {% set temp_table_name = elementary.table_name_with_suffix(test_name, "__" ~ table_type) %}
         {{ elementary.debug_log(table_type ~ ' table: ' ~ database_name ~ '.' ~ schema_name ~ '.' ~ temp_table_name) }}
 
+        {#
```
Comment (elongl) [inline macros/edr/tests/test_utils/create_elementary_test_table.sql:6]: Why is this needed if we use `CREATE OR REPLACE` to create the test tables?

**Comment by IDoneShaveIt** [comment]
There was no bug, It was an orchestration problem when upgrading elementary
